x/amm/types/pool.go

```diff
@@ -88,6 +88,7 @@ func NewPoolState(tick int32, price sdk.Dec) PoolState {
 		CurrentTick:                tick,
 		CurrentPrice:               price,
 		CurrentLiquidity:           utils.ZeroInt,
+		TotalLiquidity:             utils.ZeroInt,
```

Comment:
In the V5 Upgrade Handler, some fields are initialized with empty and zero states through the `NewPoolState` and then migrated for each liquidity, so it would be good to add a detailed case to `TestUpgradeV5` to see if the `PoolState` is set well to the expected value, just in case, for safety verification including `TotalLiquidity`

Reply:
The zero default values are same in `CreatePool`, but I'll add some checks as you said.